images/openshift-enterprise-service-idler.yml

```diff
@@ -1,11 +1,12 @@
+mode: disabled
```

Comment:
uh... don't do that?
i see that https://github.com/openshift-priv/service-idler is broken; let's make a note to fix the fast-forwarding there, and then NOT break the build in the meantime :) (leave as non-priv org)